packages/card/locales/da/messages.po

```diff
@@ -0,0 +1,20 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2024-07-30 14:45+0200\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: da\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
+
+#. Screenreader message to indicate that the card is clickable
+#. js-lingui-explicit-id
+#: packages/card/index.js:36
+msgid "card.button.text"
+msgstr ""
```

Comment:
Just a reminder that we'll need to wait for official translations before we can make a stable release. 
Maybe we could suggest a translation for the `Select` message in Danish and Finnish so we are not blocked, and if those are incorrect, we'll get a Crowdin PR afterwards. At least there will be a translation available. A quick search suggests these:
Danish: "Vælg"
Finnish: "Valitse"

Reply:
Good idea! 🙌

Reply:
Updated in this commit: [ab72db0](https://github.com/warp-ds/elements/pull/177/commits/ab72db064323acb00daf8cf2d23036e395021ee8)